home-manager/home-manager

```diff
@@ -309,7 +309,48 @@ $xdgVars
   # You can update Home Manager without changing this value. See
   # the Home Manager release notes for a list of state version
   # changes in each release.
-  home.stateVersion = "22.11";
+  home.stateVersion = "22.11";  # Read the comment before changing!
```

Comment:
Maybe a bit patronising?

Reply:
Fair enough 🙂 I changed it to something a bit more polite.